### PR TITLE
Add wrapper for bax2bam

### PR DESCRIPTION
**Pull request by stephanflemming**
summary:
- add wrapper for bax2bam, a tool that converts PacBio basecall format (bax.h5) into BAM
- Closes: #2613

notes:
- github: https://github.com/pacificbiosciences/bax2bam/
- parameter overview: https://docs.google.com/spreadsheets/d/1-31o1U1jenrKIsGKM-ojJj0IVNmB27UwzSjyAIzFi9Q/edit?usp=sharing

### CODE REVIEW

File: tools/pacbio/bax2bam.xml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    </macros>
+
+    <expand macro="requirements"/>
+
+    <stdio></stdio>
+
+    <command detect_errors="exit_code"><![CDATA[
+        bax2bam
+            #for $file in $files
+                $file
```
Comment (bgruening) [inline tools/pacbio/bax2bam.xml]: this needs to be single-quoted

File: tools/pacbio/bax2bam.xml [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    <configfiles>
+    </configfiles>
+
+    <inputs>
+        <param name="files" type="data" format="h5" multiple="true" label="Files" help="Should be from the same movie."/>
+        <param name="readtype" type="select" label="Output read type" help="">
+            <option value="--subread">subread</option>
+            <option value="--hqregion">hqregion</option>
+            <option value="--polymeraseread">polymeraseread</option>
+            <option value="--ccs">ccs</option>
```
Comment (bgruening) [inline tools/pacbio/bax2bam.xml]: but this means that if you select -ccs no output is generated right?

Comment (stephanflemming) [reply tools/pacbio/bax2bam.xml]: Removed.

File: tools/pacbio/bax2bam.xml (hunk 1 above)
Comment (stephanflemming) [reply tools/pacbio/bax2bam.xml]: Done.

**Comment by bgruening** [comment]
Thanks @stephanflemming! Please ask Simon how to add those tools to our Galaxy instance tomorrow.

**Pull request merged**